exp-player/addon/components/exp-player.js

```diff
@@ -58,7 +58,10 @@ export default Ember.Component.extend(FullScreen, {
                 expData: this.get('expData'),
                 sequence: sequence
             };
-            this.sendAction('saveHandler', payload);  // call the passed-in action with payload
+            var exitUrl = this.get('experiment.exitUrl');
+            this.sendAction('saveHandler', payload,  () => {
```

Comment:
Can you please clarify what the third argument does? The documentation suggests there are only two parameters.
http://emberjs.com/api/classes/Ember.Component.html#method_sendAction

Since this is a somewhat route-related concern, you may wish for the window.location (or `this.transitionTo(urlish)`) call to not live in the player.